app/models/tiny_mce_asset.rb

```diff
@@ -21,20 +28,55 @@ class TinyMceAsset < ApplicationRecord
   validates_attachment :image,
                        presence: true,
                        size: {
-                         less_than: Rails.configuration.x.file_max_size_mb.megabytes
+                         less_than: Rails.configuration.x\
```

Comment:
is it correct?

Reply:
Yeah, \ is like continue on next line.

"a" \
"b" \
"c"
=> "abc"